### PR TITLE
ref(feedback): always display feedback details if from alert

### PR DESCRIPTION
**Pull request by michellewzhang**
closes https://github.com/getsentry/team-replay/issues/285

- will change the project selector to include the new project, if it isn't already selected, in a following PR

### CODE REVIEW

File: static/app/components/feedback/useFeedbackOnboarding.tsx [hunk 1]
```diff
@@ -45,3 +46,10 @@ export function useHaveSelectedProjectsSetupFeedback() {
     fetching,
   };
 }
+
+export function useFeedbackFromSlack() {
+  const {getParamValue: getReferrer} = useUrlParams('referrer', '');
+  const isFromSlack = getReferrer() === 'slack';
```
Comment (JoshFerge) [inline static/app/components/feedback/useFeedbackOnboarding.tsx:52]: what if its from an email?

Comment (michellewzhang) [reply static/app/components/feedback/useFeedbackOnboarding.tsx:52]: we can also apply this to emails. do you know the referrer name for that?

Comment (michellewzhang) [reply static/app/components/feedback/useFeedbackOnboarding.tsx:52]: actually, nvm i can figure it out from the emails i've gotten!

File: static/app/views/feedback/feedbackListPage.tsx [hunk 2]
```diff
@@ -72,7 +76,7 @@ export default function FeedbackListPage({}: Props) {
             <ErrorBoundary>
               <LayoutGrid>
                 <FeedbackFilters style={{gridArea: 'filters'}} />
-                {hasSetupOneFeedback ? (
+                {hasSetupOneFeedback || isFromSlack ? (
```
Comment (JoshFerge) [inline static/app/views/feedback/feedbackListPage.tsx:79]: i feel like we should just look to see if the link has a feedback slug, and if so, always show the feedback

Comment (michellewzhang) [reply static/app/views/feedback/feedbackListPage.tsx:79]: that works, i'll change this

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**